eng/common/TestResources/New-TestResources.ps1

```diff
@@ -109,7 +109,7 @@ trap {
 }
 
 # Enumerate test resources to deploy. Fail if none found.
-$root = [System.IO.Path]::Combine("$PSScriptRoot/../sdk", $ServiceDirectory) | Resolve-Path
+$root = [System.IO.Path]::Combine("$PSScriptRoot/../../../sdk", $ServiceDirectory) | Resolve-Path
```

Comment:
This was already fixed in azure-sdk-tools and merged into the repo. I forgot to rebase this on master but will effectively no-op when merged (not a conflict).